src/components/EventForm.tsx

```diff
@@ -200,21 +235,30 @@ const EventForm = ({
                   }
                 }}
                 tileClassName={({ activeStartDate, date, view }) => {
-                  if (Date.now() > date.getTime()) {
+                  const today = new Date()
+                  today.setHours(0, 0, 0, 0)
+                  if (
+                    // Disable past dates and days past 60 days of today
+                    today.getTime() > date.getTime() ||
+                    new Date(
+                      new Date().setDate(new Date().getDate() + 60),
```

Comment:
should 60 be configurable? looks like this and 7 get used in a few places and should be configurable since we might want to support more.